client/lib/domains/constants.js

```diff
@@ -1,34 +1,28 @@
 /** @format */
 
-/**
- * External dependencies
- */
-
-import keyMirror from 'key-mirror';
-
-const type = keyMirror( {
-	MAPPED: null,
-	REGISTERED: null,
-	SITE_REDIRECT: null,
-	WPCOM: null,
-	TRANSFER: null,
-} );
+export const type = {
```

Comment:
Any reason why `key-mirror` is _passé_ now? 😀 I don't have any strong feelings about it, but it did save some duplication and removed the chance of making a typo in the key's value. Like, `CANCELED: 'CANCELLED'` - see the problem?

Reply:
yeah I just fail to see its value to us. first of all, I'm pretty sure it's bloating our bundle size on account of the fact that it introduces another import into our module and at this moment those imports are disproportionately heavy to how they look. secondly, `keyMirror` saves a little typing but not much. those duplicate strings get compressed very well, plus we can even inline them and prune out the constant imports (check out #21015 for inspiration).

I'll admit it increases the risk of mistyping those things, but then again, I don't think that matters. as constants we're referencing them by their symbol, not their value. I think the only real risk we have of breakage is if we type the same value in for two distinct keys.